docs/02.API-REFERENCE.md

```diff
@@ -3868,6 +3868,121 @@ example (void)
 - [jerry_create_promise](#jerry_create_promise)
 - [jerry_promise_state_t](#jerry_promise_state_t)
 
+## jerry_from_property_descriptor
+
+**Summary**
+
+*Notes*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+- The es.next profile enables this by default.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_from_property_descriptor (const jerry_property_descriptor_t *src_prop_desc_p)
+```
+
+- `src_prop_desc_p` - the input property descriptor.
+- return
+  - [jerry_value_t](#jerry_value_t)
+  - jerry_value_t - if success
+  - value marked with error flag - otherwise
+
+**Example**
+
+[doctest]: # (test="compile")
+
+```c
+#include <jerryscript.h>
+
+static void
+example (void)
+{
+  // create a object.
+  jerry_value_t object = jerry_create_object ();
+  // define a string
+  jerry_value_t prop_name = jerry_create_string_from_utf8 ((jerry_char_t *) "value");
+  // create a value to the 'value' property
+  jerry_value_t value = jerry_create_boolean (true);
+
+  // setting 'value' property with the value true on the object variable
+  jerry_set_property (object, prop_name, prop_name);
+
+  // define a property_descriptor
+  jerry_property_descriptor_t prop_desc;
```

Comment:
Property descriptor field must be inited.

Reply:
This request is still valid.

Reply:
Just initialize the descriptor from C, rather than creating from JS object.